packages/suite/src/views/wallet/sign-verify/index.tsx

```diff
@@ -109,6 +109,10 @@ const tooltipContent = (
     />
 );
 
+const MultilineRow = styled(Row)`
+    align-items: start;
+`;
+
```

Comment:
i believe it's better to remove the top margin [here](https://github.com/trezor/trezor-suite/blob/642239ca4441c36d60284254e2aff79a1079f398/packages/suite/src/views/wallet/sign-verify/index.tsx#L78), as it's a leftover measure for the previous version of the `SelectBar`